ops/docker/op-stack-go/Dockerfile.dockerignore

```diff
@@ -19,6 +19,7 @@
 !/op-service
 !/op-supervisor
 !/op-test-sequencer
+!/op-e2e
```

Comment:
Why are we making this change? Do we build any docker images using cod which depends on `op-e2e`?

Reply:
I gather op-test-sequencer build depends on it.

Reply:
There is an opportunity to optimize here since op-e2e is 6.8MB whereas op-e2e/e2eutils is only 320KB.